homeassistant/components/telnet/switch.py

```diff
@@ -96,82 +98,71 @@ class TelnetSwitch(SwitchEntity):
 
     def __init__(
         self,
-        hass,
-        object_id,
-        resource,
-        port,
-        friendly_name,
-        command_on,
-        command_off,
-        command_state,
-        value_template,
-        timeout,
-    ):
+        hass: HomeAssistant,
```

Comment:
Don't pass in `hass`. It will be set on the entity when the entity has been added to home assistant. This means you can access `hass` with `self.hass`, after the entity has been added to home assistant.